### PR TITLE
eclipse me!!

### PR DESCRIPTION
**Pull request by Just-a-Unity-Dev**
written on mobile because my pc is broke and im sick as fuuck

### CODE REVIEW

File: domains/eclipse.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "owner": {
+        "username": "Just-a-Unity-Dev",
+        "email": "",
+        "twitter": "@eclips_e69"
+    },
+    "description": "A little portfolio based with a basic account system for learning purposes.",
+    "record": {
+        "CNAME": "eclips-espace-production.up.railway.app"
```
Comment (MaskDuck) [inline domains/eclipse.json:9]: application error?

**Review by MaskDuck — CHANGES_REQUESTED** [review]

File: domains/eclipse.json (hunk 1 above)
Comment (Just-a-Unity-Dev) [reply domains/eclipse.json:9]: whoops, accidentally made railway shit itself. i'll try to fix it later with my mothers laptop

Comment (MaskDuck) [reply domains/eclipse.json:9]: oops

**Comment by Just-a-Unity-Dev** [comment]
@MaskDuck i fixed the railway issue, can you take a look at it?

**Comment by Just-a-Unity-Dev** [comment]
@MaskDuck i fixed the railway issue, can you take a look at it?

**Review by MaskDuck — APPROVED** [review]

**Pull request merged**